api/data_explorer/__main__.py

```diff
@@ -21,28 +21,24 @@
 # values. These arguments will rarely be specified as flags directly, aside from
 # occasional use during local debugging.
 parser = argparse.ArgumentParser()
-parser.add_argument(
-    '--path_prefix',
-    type=str,
-    help='Path prefix, e.g. /api, to serve from',
-    default=os.environ.get('PATH_PREFIX'))
-parser.add_argument(
-    '--elasticsearch_url',
-    type=str,
-    help='Elasticsearch url, e.g. elasticsearch:9200',
-    default=os.environ.get('ELASTICSEARCH_URL'))
-parser.add_argument(
-    '--dataset_config_dir',
-    type=str,
-    help='Dataset config dir. Can be relative or absolute',
-    default=os.environ.get('DATASET_CONFIG_DIR'))
+parser.add_argument('--path_prefix',
```

Comment:
nit for PR name -- There's "selected facet values" and "extra facets", not sure what "selected facets" is